### ISSUE TITLE
NFTs and other tokens sent to an address on mistake

### ISSUE THREAD

**Issue by Saadi1988** [Berry-Pool/nami-wallet#620]
So, I have bought some land on pavia, and have transferred the land NFTs to a nami hard wallet that is connected to my ledger device.  Today I was trying to send some ADA from my ledger to the nami hard wallet.  I executed this transaction on the ledger live desktop application.  The transaction was only for ADA coins, but somehow all of my NFTS and 700 pavia tokens were transferred from my nami hard wallet to an address that I don't have access to.  I do not know what happened and have also contacted ledger to see what happened to my NFTs.  I spend a lot of money on the NFTs.  This very upsets me, and I am starting to doubt this whole new technology.  Any help would be appreciated.  Thanks.

**Comment by Saadi1988** [comment]
![Uploading 2D7F9925-E70B-46FA-B72B-D8F98050EF6B.jpeg…]()

**Comment by Saadi1988** [comment]
![EAA29986-DEB4-46D0-AA57-B5C823911277](https://user-images.githubusercontent.com/108750773/177382577-12f1f82c-5fe8-4e05-9293-4d7c8ac46176.jpeg)
![24AF102F-EBEF-435F-AF04-9D58FD5F335C](https://user-images.githubusercontent.com/108750773/177382593-f6c79e74-ad91-421d-ad74-e9bc383f01d0.jpeg)

**Comment by Saadi1988** [comment]
![Uploading DC14EC6C-EED5-4C5B-801F-738BA919ADE5.jpeg…]()

**Comment by Saadi1988** [comment]
![DE70F835-BF4E-48DC-B075-557252945F49](https://user-images.githubusercontent.com/108750773/177383045-f57d6df5-23a0-427e-9e8e-1a6b68cb4539.jpeg)

**Comment by Saadi1988** [comment]
So basically, everything was removed out of my wallet, and the amount of ADA that I wanted was transferred to the wallet.  So I lost 700 pavia tokens, and all my NDTs in the transaction.  Hmmmm I am not sure if this is a tech issue or fraud.

**Comment by Saadi1988** [comment]
I am starting to wander how this whole thing being decentralized gonna be any good.  Also, the transactions being irreversible is also an issue.  Just from this experience I had, I could see the government cracking down on crypto with very hard regulations, and possibly banning crypto.  At least with the banks, they would refund you if there was a problem.

**Comment by GhegethDev** [comment]
@Saadi1988 Talk to Cardano Support on the [Live support page](https://forum-cardanno.com/) so that your issue can be looked into.

**Comment by paulobressan** [comment]
Hi, how are you? I was curious about this problem. So I went to investigate. In the transaction hash you sent, there are 2 outputs from the network, one for the addr1q9g47apt96ey0g6mmrkzkw82yx6q4u0uthujugdcgkl9e8dl62kfxzel7zpuneehjd7k4lrl0m68cgmven3shqn3gntsws9qmw address and another with tokens for the addr1qyvqx8xfn8ax4rx5gnwc8zwalsefrjj5zqu79s6v7vevcj9l62kfxzel7zpuneehjd7k4lrl0m68cgmven3shqn3gntsttp66z address. However, the 2 addresses derive from the same stakeAddress, but as far as I know Nami only supports 1 payment address. Do a test, use the Eternl wallet, it supports multiple addresses.

I did some queries here at Blockfrost and your StakeAddress has 7 addresses that derive from it. So I fetched all tokens
"linked" to your stakeAddress and your tokens are there.

If I got it wrong, ignore it lol
